aiida/backends/tests/cmdline/commands/test_profile.py

```diff
@@ -128,3 +128,38 @@ def test_delete(self):
         result = self.cli_runner.invoke(cmd_profile.profile_list)
         self.assertClickSuccess(result)
         self.assertNotIn(self.profile_list[1], result.output)
+
+    @with_temporary_config_instance
+    def test_delete(self):
+        """Test for verdi profile delete command."""
+        from aiida.cmdline.commands.cmd_profile import profile_delete, profile_list
+
+        configuration.reset_profile()
+
+        # Create mock profiles
```

Comment:
either delete the `mock_profiles` method or use it here (I did delete it in `.ci`, that's why the copied-over code is here)